quantum/nodes/script/script.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const shell = require('../../python').PythonShell;
+
+module.exports = function(RED) {
+  function ScriptNode(config) {
+    RED.nodes.createNode(this, config);
+
```

Comment:
Should include a 'waiting' phase similar to the `simulator` node.

Reply:
Thanks Theo, will take a look into this later today